Menu for changing Colors/Saber Override In game

Kylemc1413 opened the pull request.

Kylemc1413 committed:
Add Settings menu for Adjusting Modprefs Values

Kylemc1413 committed:
Add Settings menu for Adjusting Modprefs Values

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
Add Settings menu for Adjusting Modprefs Values

Kylemc1413 committed:
Added setting for custom saber override to menu

Kylemc1413 committed:
Setting for Saber Trails if SaberTailor is Installed

Kylemc1413 commented:
Known issue caused by how base plugin works, Song lighting won't reflect further changes to colors after a song is played once

SteffanDonal commented:
Cheers for this - will test.

I would rather keep the settings panel for SaberTailor separate code-wise, however!

Kylemc1413 commented:
I kinda just threw the trail option on the left color menu for now since it only shows up if saber tailor is installed and I plan on restructuring the menu once I add color presets and an option to change the increment for changing the menu values

brian91292 committed:
Fix ingame lights when colors are changed using the new ingame settings interface

Kylemc1413 committed:
Merge pull request #1 from brian91292/master

Fix in game lighting lights when colors are changed from the settings menu

Kylemc1413 committed:
Color Presets, Increment, Menu Restructure

Kylemc1413 committed:
Update CustomColors.csproj

brian91292 committed:
Merge pull request #1 from Kylemc1413/master

update

brian91292 committed:
Less ghettoness

-ScriptableColors now get set by name, so it's always consistent
-Menu text colors also get set by name now, more consistent
-Make sure prepass lights don't get double set, still ghetto but seems to be working :thumbsup:

Kylemc1413 committed:
Merge pull request #2 from brian91292/master

Attempt to fix bug that caused right/left saber colors to swap inconsistently

brian91292 committed:
Huge code cleanup

-Added new component type: IntViewController, used to toggle the value of ints in the settings menu

Kylemc1413 committed:
Merge pull request #3 from brian91292/master

Huge code cleanup

brian91292 committed:
Add userIncrement to the menu

Kylemc1413 committed:
Update readme

Kylemc1413 committed:
Merge pull request #4 from brian91292/master

Add userIncrement to the menu

Kylemc1413 committed:
Fix conversion of user colors to float

brian91292 committed:
fixes

-No longer uses userIncrement globally for any IntViewController component, only uses userIncrement for right/left rgb colors.
-Revert stupid things I did that I should have tested more thouroughly

Kylemc1413 committed:
Merge pull request #5 from brian91292/master

fixes

brian91292 committed:
custom colored walls!

brian91292 committed:
changed menu option to customWallColor instead of wallsUseLeftColor

custom wall color can now be completely disabled from the menu

Kylemc1413 committed:
Merge pull request #6 from brian91292/master

custom colored walls!

Kylemc1413 committed:
Wall Color Presets, Try To Lower Wall Brightness

Kylemc1413 committed:
Up opacity of walls slightly

Kylemc1413 committed:
Fix look of colored walls

Kylemc1413 committed:
Add Presets for Lighting,  Settings Restructure

Kylemc1413 committed:
Update Readme

Kylemc1413 committed:
Disable plugin if ChromaToggle installed to avoid conflict

brian91292 committed:
Merge pull request #2 from Kylemc1413/master

update to kyles fork

brian91292 committed:
fixed crash when null material is referenced

Kylemc1413 committed:
Merge pull request #7 from brian91292/master

fix crash when null material is referenced within a custom saber

Kylemc1413 committed:
Brightness setting, Pure black preset

Kylemc1413 committed:
Merge branch 'master' of https://github.com/Kylemc1413/BeatSaber-CustomColors

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
Remove ui temporarily

Kylemc1413 committed:
Update CustomColors.csproj

Kylemc1413 committed:
Adjust lighting

Kylemc1413 committed:
Adjustments

Kylemc1413 committed:
Still ignoring ui for now

Kylemc1413 committed:
Update SettingsUI.cs

Kylemc1413 committed:
Ui

Kylemc1413 committed:
UI

Kylemc1413 committed:
Update readme.md

Kylemc1413 committed:
Update readme.md

Kylemc1413 committed:
Update readme.md

Kylemc1413 committed:
Namespacing for BeatSaberCustomUI

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
Merge branch 'master' of https://github.com/Kylemc1413/BeatSaber-CustomColors

Kylemc1413 committed:
Move color changing to scenechange

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
1.6.0

Kylemc1413 committed:
Fix colors being incorrect

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
Merge prepasslights fix from brian91292

Kylemc1413 committed:
Version

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
RainbowWall setting

Kylemc1413 committed:
1.6.8

Kylemc1413 committed:
Option to Disable the plugin

Kylemc1413 committed:
Version

Kylemc1413 committed:
Fix Disable

Kylemc1413 committed:
Setting to allow color overrides

Kylemc1413 committed:
New Presets

Kylemc1413 committed:
Klouder blue preset, small fix

Kylemc1413 committed:
Version

Kylemc1413 committed:
Cleanup, FINALLY get colors changing nicely?

Kylemc1413 committed:
Scratch that, NOW FEEX

Kylemc1413 committed:
Fix custom saber color overrides using Custom Colors

Kylemc1413 committed:
Override Logic Galore

Kylemc1413 committed:
Update CustomColors.csproj

Kylemc1413 committed:
Update Plugin.cs

Kylemc1413 committed:
Version

Kylemc1413 committed:
Miku, Feex

Kylemc1413 committed:
Update Plugin.cs

brian91292 committed:
fixed wall performance issues

Kylemc1413 committed:
Merge pull request #9 from brian91292/master

fixed wall performance issues

Kylemc1413 committed:
Improve Rainbow Walls, Finish with new non laggy wall color changing setup

Kylemc1413 committed:
Fix

Kylemc1413 committed:
Fix color overrides in one saber

Kylemc1413 committed:
Revert "Fix color overrides in one saber"

This reverts commit 1528b138a6ad040f732c00c01f66b9b230da6d47.

Kylemc1413 committed:
Actually fix one saber overrides

Kylemc1413 committed:
Add function for mods to override custom saber colors if user settings permit

Kylemc1413 committed:
Optimization, version

Kylemc1413 committed:
1.10.9

Kylemc1413 committed:
New preset, minor fixes and clearing out dead code

Kylemc1413 committed:
Update AssemblyInfo.cs

Kylemc1413 committed:
.

Kylemc1413 committed:
0.13.0 Update. Cleanup

Kylemc1413 committed:
Fix light breaking on applying settings
